### DISCUSSION TITLE
Is there any protocol specification that rustdesk used?

### DISCUSSION THREAD

**Discussion by open-cmi** [rustdesk/rustdesk#7011] [General]
I am doing a protocol recognize work, and I want to know the rustdesk protocol specification. Is there any document about protocol specification that rustdesk used?

**Comment by xlionjuan** [comment]
https://github.com/rustdesk/rustdesk/wiki/How-does-RustDesk-work%3F

**Reply by open-cmi** [reply]
Thank you!
